src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java

```diff
@@ -101,10 +101,22 @@ public String getCommitSha() {
         return branchDetails.getCommitId();
     }
 
+    public String getDashboardUrl() {
+        return publicRootURL + "/dashboard?id=" + URLEncoder.encode(project.getKey()) + "&pullRequest=" + branchDetails.getBranchName();
+    }
+
+    public String getIssueUrl(String issueKey) {
+        return publicRootURL + "/project/issues?id=" + URLEncoder.encode(project.getKey()) + "&pullRequest=" + branchDetails.getBranchName() + "&issues=" + issueKey + "&open=" + issueKey;
+    }
+
     public QualityGate.Status getQualityGateStatus() {
         return qualityGate.getStatus();
     }
 
+    public Map<RuleType, Long> getRulesCount() {
+        return countRuleByType();
```

Comment:
We not just make `countRuleByType` public, rather than introduce a method that just delegated directly to it?

Reply:
No, I have reverted the change